### PR TITLE
support new recaptcha version

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: card/db/migrate_core_cards/20150914134339_admin_info_card.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    Card[:recaptcha_settings].update_attributes! :type_id=>Card::PointerID,
+      :content=>"[[+public key]]\n[[+private key]]\n[[+proxy]]"
+
+    ['public_key', 'private_key', 'proxy'].each do |name|
+      card = Card["recaptcha_#{name}".to_sym]
+      card.update_attributes! :name=>"#{Card[:recaptcha_settings].name}+#{name.gsub('_',' ')}"
+    end
+    Card.fetch('*recaptcha settings+*self+*structure').delete
+
```
Comment (ethn) [inline card/db/migrate_core_cards/20150914134339_admin_info_card.rb]: We need to delete the *script output (if not build a system to detect file updates; probably not yet, given the time crunch...)

File: card/mod/04_settings/set/self/recaptcha_public_key.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+event :update_recaptcha_public_key, :after=>:store do
```
Comment (ethn) [inline card/mod/04_settings/set/self/recaptcha_public_key.rb]: I'm pretty confused about these new configurations.  Are they Settings or not?  In the migration, it looks like they're Phrase cards, but here we're using `extend Card::Setting`.  A recaptcha public key isn't set-specific, is it?

If not, why do we assign them to groups?

File: wagn/lib/wagn/generators/wagn/templates/config/application.rb [hunk 3]
```diff
@@ -8,8 +8,9 @@ class Application < Wagn::Application
 
     # Wagn inherits Rails configuration options.  See http://guides.rubyonrails.org/configuring.html
 
-    config.recaptcha_public_key  = '6LdhRssSAAAAAFfLt1Wkw43hoaA8RTIgso9-tvtc'
-    config.recaptcha_private_key = '6LdhRssSAAAAAGwzl069pJQBdmzCZigm1nV-dmqK'
+    config.recaptcha_public_key  = '6LeoHfESAAAAAN1NdQeYHREq4jTSQhu1foEzv6KC'
+    config.recaptcha_private_key = '6LeoHfESAAAAAHLZpn7ijrO4_KGLEr2nGL4qjjis'
```
Comment (ethn) [inline wagn/lib/wagn/generators/wagn/templates/config/application.rb]: If we're migrating these into cards anyway, shouldn't we just make these the default card content and get rid of the code configuration option?

**Comment by ethn** [comment]
Do I understand correctly that this follows the same interface pattern as our current one; it just uses the new api?

**Comment by xithan** [comment]
The interface pattern didn't change. You click on submit and then you get the recaptcha.

**Pull request merged**